risc0/zkvm/Cargo.toml

```diff
@@ -148,20 +165,25 @@ prove = [
   "dep:protoc-prebuilt",
   "dep:rayon",
   "dep:rustc-demangle",
-  "dep:sha2",
   "dep:tempfile",
   "dep:typetag",
   "risc0-circuit-recursion/prove",
   "risc0-circuit-rv32im/prove",
   "risc0-zkp/prove",
   "std",
 ]
-std = [
-  "anyhow/std",
-  "num-traits?/std",
-  "risc0-binfmt/std",
+std-verify = [
+  "ark-bn254/std",
+  "ark-groth16/std",
+  "ark-serialize/std",
+  "num-bigint/std",
+  "num-traits/std",
   "risc0-circuit-recursion/std",
   "risc0-circuit-rv32im/std",
+  "sha2/std"
+]
```

Comment:
I don't think we need this.

We should be able to do:

```
std = ["ark-bn254?/std", ...]
```

Reply:
If we do this though, the `?` doesn't really seem to work. It will import every dep specified within the feature

Reply:
The cargo docs say:
> The "package-name/feature-name" syntax will also enable package-name if it is an optional dependency. Often this is not what you want. You can add a ? as in "package-name?/feature-name" which will only enable the given feature if something else enables the optional dependency.

Reply:
I don't think we can use `Cargo.lock` as a guide to determine if it's working or not, because this is on a per-workspace basis and is the aggregate of all `Cargo.toml` files in the workspace.